.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: go
+go:
+  - 1.6.3
+  - 1.7
+sudo: required
+dist: trusty
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install fglrx opencl-headers nvidia-opencl-dev
```

Comment:
Is fglrx actually necessary here?  I don't recall having to install it on my machine.